applications/ConvectionDiffusionApplication/python_scripts/adjoint_diffusion_solver.py

```diff
@@ -9,7 +9,6 @@ def CreateSolver(model, settings):
 class AdjointDiffusionSolver(PythonSolver):
 
     def __init__(self, model, custom_settings):
-        self._validate_settings_in_baseclass=True # for backwards compatibility, to be removed eventually
         super(AdjointDiffusionSolver,self).__init__(model,custom_settings)
```

Comment:
```suggestion
        super().__init__(model,custom_settings)
```

since you are already modifying it ;)